ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -101,15 +101,7 @@ const primalOutputStrings = {
     ko: '${text} (${junction})',
   },
   // Junctions.
-  'spread': {
-    // Shiva spread.
-    en: 'spread',
-    de: 'verteilen',
-    fr: 'dispersion',
-    ja: '散開',
-    cn: '散开',
-    ko: '산개',
-  },
+  'spread': Outputs.spread,
```

Comment:
this is lower-case, mostly because it's in parentheses and meant to be lower priority.